src/main/java/com/gluonhq/substrate/util/Logger.java

```diff
@@ -55,7 +55,7 @@ public static void logFinest(String s) {
     public static void logSevere(String s) {
         LOGGER.severe(s + "\n" +
                 "Check the log files under "+logDirectory +
-                "\nAnd please check https://docs.gluonhq.com/client/ for more information.");
+                "\nAnd please check https://docs.gluonhq.com/ for more information.");
```

Comment:
Do we want to add this: https://docs.gluonhq.com/#_gluonfx_plugin_for_maven ?

Reply:
Ah, this is common for both plugins, it would be weird to get this log in Gradle plugin.